argo-cd-apps/base/all-clusters/infra-deployments/kustomization.yaml

```diff
@@ -11,5 +11,6 @@ resources:
   - disable-csvcopy-for-all-cluster
   - enable-dvo-for-all-cluster
   - backup
+  - kubesaw-common
```

Comment:
Adding this application here is OK as we want it in staging and production but we should not deploy this in development overlay. Currently, host and member application exists in dev but they point to a folder that contains nothing which is not the case for kubesaw-common.
I opened this PR(https://github.com/redhat-appstudio/infra-deployments/pull/3576) to remove host and member from development overlay so we should get it merge, rebase this PR and delete kubesaw-common from dev overlay

Reply:
OK waiting for PR(https://github.com/redhat-appstudio/infra-deployments/pull/3576) to merge. I assume I will remove kubesaw-common from dev overlay in another PR right?

Reply:
In this PR, In fact you can do it without waiting for my PR to go in, just follow the example from my PR

Reply:
My PR passed the tests, can you approve and lgtm it, it will merge with approvals